themes-default/slim/src/components/subtitle-search.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+<!-- template for the subtitle-search component -->
+<tr class='subtitle-search-wrapper'>
+    <td colspan='9999' transition="expand">
```

Comment:
2. It's the only of having the td behaving in the <tr> full with. If I don't set it, it will limit the with the episodes table first column.

Moving the hack to a parent node, doesn't full any less hacky to me, tbh? It's more a CSS problem. I don't want to make it a `vue problem`.